NuGet.config

```diff
@@ -0,0 +1,7 @@
+<?xml version="1.0" encoding="utf-8"?>
+<configuration>
+  <packageSources>
+    <clear/>
+    <add key="Internal Analyzers" value="https://securitytools.pkgs.visualstudio.com/_packaging/SecureDevelopmentTools@Local/nuget/v3/index.json" />
+  </packageSources>
```

Comment:
From the error I see in the build here:
"Unable to find package MSBuild.Sdk.Extras. No packages exist with this id in source(s): Internal Analyzers"
it looks like the nuget task is only going to Internal Analyzers to source the nuget packages. I think you may want to add an entry to the default nuget feed because that is where the MSBuild.Sdk.Extras lives.

"https://api.nuget.org/v3/index.json" is the source

Reply:
Yes, I added that before but as Jean-Marc mentioned in the standup, we cannot include internal and external feeds in the nuget.config. We need to create an upstream for nuget.org and add it to the artifacts.

Reply:
Refer to: https://docs.opensource.microsoft.com/tools/cg/security-alerts/nuget-multifeed-configuration.html#internal-projects-eg-non-oss

Reply:
Basically it suggests using nuget.org as upstream: https://docs.microsoft.com/en-us/azure/devops/artifacts/nuget/upstream-sources?view=azure-devops